src/common/System.cpp

```diff
@@ -278,7 +278,7 @@ void SetupCrashHandler()
     nacl_minidump_set_callback(CrashHandler);
 }
 #else
-static void CrashHandler(int sig)
+[[noreturn]] static void CrashHandler(int sig)
```

Comment:
we have `NORETURN` instead. Either it isn't needed anymore, and it should probably be replaced, or it is still needed, and you would need to use it.